README.md

```diff
@@ -0,0 +1,75 @@
+[![Build Status](https://travis-ci.org/kuzzleio/sdk-java.svg?branch=master)](https://travis-ci.org/kuzzleio/sdk-java)
+
+Official Kuzzle JAVA SDK
+======
+
+## About Kuzzle
+
+A backend software, self-hostable and ready to use to power modern apps.
+
+You can access the Kuzzle repository on [Github](https://github.com/kuzzleio/kuzzle)
+
+## SDK Documentation
+
+The complete SDK documentation is available [here](http://docs.kuzzle.io/sdk-reference/)
+
+## Protocol used
+
+The JAVA SDK implement the websocket protocol.
+
+## Build
+
+### Clone the depot
+
+```sh
+git clone --recursive git@github.com:kuzzleio/sdk-java.git
+cd sdk-java
+git submodule update --init --recursive
+make
+```
+
+You should know have the SDK in the `build/java/build/libs` directory.
+
+## Installation
+
+### Bintray repository
+
+You can find the sdk's for x86 and amd64 on bintray at https://bintray.com/kuzzle/maven
```

Comment:
The SDK is available for both x86 and amd64 architectures on bintray: https://bintray.com/kuzzle/maven